.github/workflows/build_hip_basic.yml

```diff
@@ -0,0 +1,53 @@
+name: Build HIP-Basic
+
+on:
+  push:
+    branches: [ develop, release/** ]
+    paths:
+      - 'HIP-Basic/**'
+      - '.github/workflows/**'
+  pull_request:
+    branches: [ develop, release/** ]
+    paths:
+      - 'HIP-Basic/**'
+      - '.github/workflows/**'
+
+env:
+    ROCM_VERSION: 6.2
+    AMDGPU_INSTALLER_VERSION: 6.2.60200-1
+
+jobs:
+    build:
+        name: "Build HIP Examples"
+        runs-on: ubuntu-latest
+        container:
+            image: ubuntu:22.04
+        steps:
+          - uses: actions/checkout@v4
+
+          - name: Install dependencies
+            run: |
+                  apt-get update -qq &&
+                  apt-get install -y build-essential g++ glslang-tools \
+                    python3 python3-pip libglfw3-dev libvulkan-dev locales wget
+                  python3 -m pip install --upgrade pip
+                  python3 -m pip install cmake
+
+          - name: Install ROCm Dev
+            run: |
+                  export DEBIAN_FRONTEND=noninteractive
+                  wget https://repo.radeon.com/amdgpu-install/${{ env.ROCM_VERSION }}/ubuntu/jammy/amdgpu-install_${{ env.AMDGPU_INSTALLER_VERSION }}_all.deb
+                  apt-get -y install ./amdgpu-install_${{ env.AMDGPU_INSTALLER_VERSION }}_all.deb &&
+                  apt-get update -qq &&
+                  apt-get -y install rocm-dev rocm-llvm-dev
+                  echo "/opt/rocm/bin" >> $GITHUB_PATH
```

Comment:
/opt/rocm/bin might not exist, particularly if you are bypassing the installer.

Reply:
/opt/rocm is managed by update-alternatives. This is needed for the  use case of wanting to have more than one version of ROCm installed at the same time on the same machine - think very large multi-user machines. /opt/rocm therefor may or may not point to the current version of ROCm that you are wanting to test.

If you are going to fall back on the "We are starting this in a fresh docker container so it will all be OK", then you need to spell out this assumption and that this code is of no value when testing the multi-version rocm use case. Otherwise it just looks like the code is wrong because you have missed this case.